ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/BackingStore/LMDB.hs

```diff
@@ -481,10 +482,19 @@ newLMDBBackingStoreInitialiser dbTracer limits sfs initFrom = do
                pure (dbsSeq, s {dbsSeq = At slot})
              Trace.traceWith dbTracer $ TDBWrite oldSlot slot
 
+           bsStat :: m (WithOrigin SlotNo, Int)
+           bsStat = do
+             guardDbClosed dbClosed
+             liftIO $ LMDB.readWriteTransaction dbEnv $ withDbStateRW dbState $ \s@DbState{dbsSeq} -> do
+               constn <- traverseLedgerTables (\(LMDBMK _ dbx) -> ConstMK <$> LMDB.size dbx) dbBackingTables
+               let n = getSum $ foldLedgerTables (Sum . getConstMK) constn
```

Comment:
This means: the sum of all the items in LMDB in all the tables. This is fine because there is only one LMDBMK table, the UTxO table. But it will not be in the future. This probably deserves a second thought. 

I also wonder, why don't we use the `mdb_stats` thingie?